@xen-orchestra/web-core/lib/components/copy-button/VtsCopyButton.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <UiButtonIcon
+    v-tooltip="copied && $t('core.copied')"
+    :icon="icon"
```

Comment:
As of Vue 3.4, you can use the shorthand here:
```suggestion
    :icon
```